src/users/services/users.service.ts

```diff
@@ -23,16 +23,16 @@ const dummy = [
     status: 'active'
   }
 ];
-
+*/
 @Injectable()
 export class UsersService {
   constructor(private http: HttpClient) { }
 
   getListOfUsers(): Observable<any> {
-    // return this.http
-    //   .get<any>(`/api/userList`)
-    //   .pipe(catchError((error: any) => throwError(error.json())));
-    return of(dummy);
+    return this.http
+       .get<any>(`/api/userList`)
+       .pipe(catchError((error: any) => throwError(error.json())));
```

Comment:
This is an antipattern in my view. Is there a good reason to catch an error then re-throw without taking any action?

As a minimum the error should be logged.

Reply:
this is angular 2+ pattern what is correct.... the errors, if we want, are then later caught by effects and reducers to update the store. 

https://angular.io/guide/http